apps/modernization-ui/src/design-system/Pagination/Pagination.tsx

```diff
@@ -10,7 +10,7 @@ const Pagination = () => {
     } = usePage();
 
     return (
-        <div className={styles.pagination}>
+        <div className={total > 0 ? styles.pagination : styles.hidden} data-testid="pagination-container-component">
```

Comment:
question:  was the styling added to preserve the height when the component is not shown?

Reply:
Kind of, I had opted for conditional rendering originally but that caused the bottom border to jump when going from no results state to loading state to results page state.